packages/slate-react/src/plugins/after.js

```diff
@@ -122,7 +122,20 @@ function AfterPlugin() {
     // Once the fake cut content has successfully been added to the clipboard,
     // delete the content in the current selection.
     window.requestAnimationFrame(() => {
-      editor.change(c => c.delete())
+      // If user cuts a void block or a void inline,
+      // manually removes them since selection is collapsed in this case.
+      const { value } = change
+      const { endBlock, endInline } = value
+      const isVoidBlock = endBlock && endBlock.isVoid
+      const isVoidInline = endInline && endInline.isVoid
+
+      if (isVoidBlock) {
```

Comment:
It seems like it would be better to use something like `if (isCollapsed && isVoid)` as the condition? Because as currently written, won't this fail to delete the full range if it happens to end in a void?

I might be missing something thought, let me know!

Reply:
Fixed!